src/app/_components/add-to-cellar-modal/index.tsx

```
@@ -0,0 +1,225 @@
+  DialogTitle,
+  DialogTrigger,
+} from "@/app/_components/ui/dialog";
+import FormError from "@/app/_components/ui/form-error";
+import type { StorageLocation } from "@/domain/storage-locations/types";
+import { usePathname } from "@/lib/i18n";
+import { cn } from "@/lib/tailwind";
+
+import type { SubmissionResult } from "@conform-to/react";
+
+// We need to import the schema from the cellar schemas
+import { addToCellarSchema } from "@/app/[locale]/(business)/(with-header)/users/[username]/cellar/schemas";
```

ESLint

[error] `@/app/[locale]/(business)/(with-header)/users/[username]/cellar/schemas` import should occur before import of `@/app/_components/form/cellar-serving-format-selector`

(import/order)